packages/components/pages/f-self-exclusion/src/components/SelfExclusion.vue

```diff
@@ -403,7 +394,11 @@ export default {
     align-items: center;
     justify-content: flex-end;
     gap: f.spacing(d);
-    padding: f.spacing(d);
+    margin-top: f.spacing(d);
+}
+
+.c-selfExclusion-buttons button {
+    margin: 0 5px 5px;
```

Comment:
this margin seems a little irregular, what was the reason for 5px? I would have expected to use a value from our spacing set

Reply:
A confirmation dialog appears on top of the button once we select one of the options. The focus of the button appears and doesn't look nice. 8px is too big and 4 is not enough.

Reply:
Anyway, it won't be visible in this format to anyone, so I just removed it.